Update directory-countries.md (#8302)
magento/devdocs: src/guides/v2.3/graphql/queries/directory-countries.md

title: countries query
---

> Use the `countries` query to return all countries in which the entity can do business. Don't Confuse with [Country]({{page.baseurl}}/graphql/queries/directory-country.html) query, its sepearate query to retrieve information about a specific country by two-letter country code.

Reviewer: ```suggestion
Use the `countries` query to return all countries in which the entity can do business. Do not confuse with [Country]({{page.baseurl}}/graphql/queries/directory-country.html) query, it is a separate query to retrieve information about a specific country by two-letter country code.
```

> Use the `countries` query to return all countries in which the entity can do business. Do not confuse with the [Country]({{page.baseurl}}/graphql/queries/directory-country.html) query, it is a separate query to retrieve information about a specific country by two-letter country code.

Reviewer 2: ```suggestion
The `countries` query returns all countries in which the entity can do business. 

Use the [country]({{page.baseurl}}/graphql/queries/directory-country.html) query if you want to retrieve information about a specific country.
```

> Use the `countries` query to return all countries in which the entity can do business. Do not confuse with the [Country]({{page.baseurl}}/graphql/queries/directory-country.html) query, it is a separate query to retrieve an information about a specific country by two-letter country code.

Reviewer: ```suggestion
Use the `countries` query to return all countries in which the entity can do business. Do not confuse with the [Country]({{page.baseurl}}/graphql/queries/directory-country.html) query, it is a separate query to retrieve information about a specific country by two-letter country code.
```

**Response:**

> In this example, the response is intentionally truncated. Here `available_regions` attribute value will be null if country doesn't have any regions available otherwise it will be an array of regions for the respective country. 

Reviewer: ```suggestion
In this example, the response is intentionally truncated. The `available_regions` attribute value will be null if the country does not have any regions available, otherwise it will be an array of regions for the respective country.
```

> In this example, the response is intentionally truncated. The `available_regions` attribute value will be null if the country does not have any regions available, otherwise it will be an array of regions for the respective country.

Reviewer 2: ```suggestion
In this example, the response is intentionally truncated. The `available_regions` attribute value will be null if the country does not have any regions available. Otherwise, it contains an array of the country's regions.
```
